### PR TITLE
Allow agda-input prefixes other than <LocalLeader>

### PR DESCRIPTION
**Pull request by CT075**
More customization options can't hurt. I find my `LocalLeader` to be irritating to hit while typing, but enough other, unrelated bindings use it that I'm not comfortable changing it wholesale.

### CODE REVIEW

File: agda-input.vim [hunk 1]
```diff
@@ -1,5 +1,6 @@
 if exists("*which_key#register")
-  inoremap <buffer> <localleader> <C-O>:WhichKey "agda"<CR>
+  let l:cornelis_agda_prefix = get(g:, "cornelis_agda_prefix", "<localleader>")
+  exec "inoremap <buffer> " . l:agda_prefix . " <C-O>:WhichKey \"agda\"<CR>"
```
Comment (isovector) [inline agda-input.vim:3]:
```suggestion
  exec "inoremap <buffer> " . l:cornelis_agda_prefix . " <C-O>:WhichKey \"agda\"<CR>"
```

?

Comment (CT075) [reply agda-input.vim]: Good catch, thanks

**Comment by isovector** [comment]
LGTM. Thanks for the patch! Happy to merge when you give the say so!

**Comment by CT075** [comment]
I think we're good!

**Pull request merged**